### DISCUSSION TITLE
can i customize Scopes and Fields by myself?

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [apache/skywalking#10622] [General]
i want to customize  Scopes and Fields and then create new oal config,however java agent Support custom enhance, but i don't sure can it customize  Scopes and Fields. thx!

**Comment by wu-sheng** [comment]
This is an open source project under Apache 2.0 License. 
I can't understand what do you mean CAN.

**Reply by qigeqigeqi** [reply]
> This is an open source project under Apache 2.0 License. I can't understand what do you mean CAN.

I want to customize the indicators used in OAL, such as business parameters.

**Reply by wu-sheng** [reply]
That is on your own. We can't reply to a thing we don't understand.
This is your work, rather than community responsibility. We never said SkyWalking is a business monitoring system.

**Reply by qigeqigeqi** [reply]
> That is on your own. We can't reply to a thing we don't understand. This is your work, rather than community responsibility. We never said SkyWalking is a business monitoring system.

all right, thx.
